tests/kernel/device/testcase.yaml

```diff
@@ -7,7 +7,7 @@ tests:
   kernel.device:
     integration_platforms:
       - native_sim
-    platform_exclude: xenvm
+    platform_exclude: xenvm xenvm/xenvm/gicv3
```

Comment:
Make all of these multiline please, this way diffs will be cleaner going forward

Reply:
Thanks for suggestion, done.